hathor/p2p/manager.py

```diff
@@ -317,11 +334,29 @@ def update_whitelist(self) -> Deferred[None]:
             settings.WHITELIST_URL.encode(),
             Headers({'User-Agent': ['hathor-core']}),
             None)
+        # Twisted Agent does not have a direct way to configure the HTTP client timeout
+        # only a TCP connection timeout. The callLater below is a manual client timeout
+        # that will cancel the deferred in case it's called
+        timeout_call = self.reactor.callLater(10, d.cancel)
```

Comment:
Move it to a constant? It's in seconds, right? I feel we should let it go for 30-45 seconds.

Reply:
Created a constant in the file and set as 45s